### DISCUSSION TITLE
Support TypeScript5

### DISCUSSION THREAD

**Discussion by NWYLZW** [vercel/community#3225] [Idea]
<img width="560" alt="image" src="https://github.com/vercel/community/assets/51358815/32cded7c-e77d-4d0e-9810-95dc1319d6f0">

<img width="588" alt="image" src="https://github.com/vercel/community/assets/51358815/681a60b3-3e80-4637-a529-d0590d41c851">

**Comment by swarnava** [comment]
Hi,

Ideally you should no longer need @vercel/node to setup your application. What framework are you using?

**Reply by NWYLZW** [reply]
I didn't use any framework, but simply used vercel's serverless function computing service.

**Reply by NWYLZW** [reply]
https://github.com/NWYLZW/25h/tree/master/api
Or I used Vite, I don't think it's a framework.
